services/github/src/handlers.js

```diff
@@ -489,17 +489,29 @@ async function statusHandler(message) {
     `Attempting to update status of the checkrun for ${organization}/${repository}@${sha} (${taskState.conclusion})`,
   );
   try {
+    const taskDefinition = await this.queueClient.task(taskId);
+    debug(`Result status. Got task build from DB and task definition for ${taskId} from Queue service`);
+    const {checkRunArtifactName} = taskDefinition.extra.github;
+
+    let checkRunArtifact = '';
+    if (checkRunArtifactName) {
+      checkRunArtifact = (await this.queueClient.getArtifact(taskId, runId, checkRunArtifactName)).toString();
```

Comment:
We should not crash the server if they have configured a `checkRunArtifactName` but the artifact did not actually get uploaded. What happens in that case with this code?

Reply:
In a non-draft PR there will be try-catch of some sort :)